src/commands/explorer/groupBy.ts

```diff
@@ -31,6 +31,5 @@ async function getQuickPicks(context: IActionContext): Promise<QuickPickItem[]>
 export enum GroupBySettings {
     ResourceGroup = 'resourceGroup',
     ResourceType = 'resourceType',
-    Location = 'location',
-    ArmTag = 'armTag'
```

Comment:
arm tag was removed because we don't want to generate the groups ahead of time?

Reply:
Yeah it being as a "default" didn't really make sense considering how we form the tag.